static/docs/user-guide/dvcignore.md

```diff
@@ -18,21 +17,22 @@ provide similar functionality as `.gitignore` files provide for `git`.
 - Each line should contain only one pattern.
 - During execution of commands that traverse directories, DVC will ignore
   matching paths.
-- Not every operation supports `.dvcignore`. To see current limitations, read
-  following paragraph.
-
-## Current limitations
-
-During development, we noticed that there are few potential uses cases that
-might be tricky to handle (e.g. what to do when we are `dvc add`-ing directory
-containing `.dvcignore` file). Therefore, we decided to enable this feature
-gradually in different parts of the project.
-
-Currently `.dvcignore` files will be read and applied in any operation that
-collects DVC-files (e.g. `checkout`, `metrics`, `status`, `run`, `repro`), so it
-is advised to use it in cases described in the first paragraph, when amount of
-files in tree of repository directory causes performance issues.
 
+## Notes
+
+- `.dvcignore` is no longer applied only on operations collecting DVC-files. 
+  Each time DVC traverses directory, dvcignore files up to project root tree 
+  are respected. 
+- ignoredd files will not be cached and preserved in any way, they will 
```

Comment:
ignored

Reply:
it needs more details on what does "preserved" means. It's not clear. For example, when we remove outputs do we skip ignored files or remove them as well? It worth describing and mentioning these edge cases. And give examples below.

Reply:
I agree, I think I should remve the preserved part. The most important thinkg about this point is that ignored files will not be cached. Ill inkclude removing edge case in new point.